HW_Python/Expl1.py

```diff
@@ -0,0 +1,17 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+# *Пример:*
+
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+n = int(input("введите трехзначное число: "))
+sum = 0
+if 99 < n < 1000:
+    while n > 0:
+        x = n % 10
+        sum = sum + x
```

Comment:
Если значение переменной используются единожды, то можно обойтись без этой переменной.
```suggestion
        sum += n % 10
```